src/config.ts

```diff
@@ -0,0 +1,84 @@
+import {cosmiconfigSync} from 'cosmiconfig';
+import {Validator} from 'jsonschema';
+import {merge} from 'lodash';
```

Comment:
Maybe we should explicitly install `lodash/merge` as a dependency if we use it in our code?